api/Element.json

```
@@ -6571,7 +6571,7 @@
             "chrome": [
               {
                 "version_added": "37",
-                "notes": "Version 92 returns a promise instead of <code>undefined</code>. The behaviour reflects this proposed specification change: https://github.com/w3c/pointerlock/pull/49."
+                "notes": "From version 92, returns a promise instead of <code>undefined</code>. The behavior reflects this proposed specification change: https://github.com/w3c/pointerlock/pull/49."
```

Throughout this PR, we ought to use a rich link, rather than a bare URL.

```suggestion
                "notes": "From version 92, returns a promise instead of <code>undefined</code>. The behavior reflects <a href='https://github.com/w3c/pointerlock/pull/49'>a proposed specification change</a>."
```